shell/platform/common/text_editing_delta.h

```diff
@@ -23,8 +23,10 @@ struct TextEditingDelta {
                    TextRange range,
                    const std::string& text);
 
+  // NOLINTNEXTLINE(google-explicit-constructor)
```

Comment:
I don't know how painful it would be, but I think these two are cases where we would want to have `explicit`.

Reply:
It worked, in this case I was confused about which one was erroring.